### ISSUE TITLE
Which distro do we support ?

### ISSUE THREAD

**Issue by Hritik14** [nexB/vulnerablecode#519]
The installation instructions mention

> System requirements
> 
>     Python 3.8+
>     PostgreSQL 9+
>     Compiler toolchain and development files for Python and PostgreSQL
> 
> On Debian-based distros, these can be installed with:
> 
> sudo apt-get install python3-venv python3-dev postgresql libpq-dev build-essential
> 

But as per https://packages.debian.org/buster/all/allpackages, all the packages listed above are already present in a debian buster installation.

We should explicitly mention which Debian we support ( and have some tests to prove that ). Supporting distros based on Debian is OK and dependencies for those should be mentioned separately.

A point to note: Our CI tests run on `ubuntu-latest` (we don't have a lot of choices there) and our Dockerfile uses `python:3.8` which on `buildpack-deps`

**Comment by pombredanne** [comment]
We should list exactly what we support, what we test on and what should be working.

**Comment by pombredanne** [comment]
we should still list the system packages we need, even if included in Buster (I would doubt postgreSQL is installed by default though)

**Comment by Hritik14** [comment]
@pombredanne Yes, postgres is installed. Have a look in the link mentioned above.

> we should still list the system packages we need

But that get out of hand pretty soon. Do we need to list Git ? svn ? what about bash ?

**Comment by pombredanne** [comment]
package-wise, everything you would install and need explicitly above an beyond the bare OS should be listed
and if we use git and svn in production, this includes them too
